simpledotorg/simple-server
# Overdue: Change order of cards and remove flipper flags

**Story card:** 
Flipper flags: [sc-10972](https://app.shortcut.com/simpledotorg/story/10972/remove-flipper-flags)
Re-ordering: [sc-10973](https://app.shortcut.com/simpledotorg/story/10973/request-to-reorder-the-overdue-cards)

## Because

Reorder the cards
- several suggestions that the Return to care card should be close to the called patients card for easier comparison. Also if the number of users/facilities is large, this card is a large distance from the other.

Remove all flipper flags
- Feature is enabled globally for all countries
- No longer requirement for flipper flags

## This addresses

- Reduce the number of flipper flags in the simple server

## Test instructions

Check that the user/facility activity card appears at the bottom of the overdue section.
Remove flipper flags from local server, test that all cards show as expected.
